wezterm-gui/src/overlay/launcher.rs

```diff
@@ -5,6 +5,7 @@
 //! be rendered as a popup/context menu if the system supports it; at the
 //! time of writing our window layer doesn't provide an API for context
 //! menus.
+use super::quickselect;
```

Comment:
I prefer to use `crate::` relative paths for this sort of import; it makes it a little easier to refactor in the future